src/zzz_od/api/routers/onedragon.py

```
@@ -0,0 +1,736 @@
+    """获取恶名狩猎的下拉框选项"""
+    from zzz_od.application.notorious_hunt.notorious_hunt_config import NotoriousHuntLevelEnum, NotoriousHuntBuffEnum
+    from zzz_od.application.battle_assistant.auto_battle_config import get_auto_battle_op_config_list
+
+    ctx = get_ctx()
+
+    # 获取任务类型选项
+    mission_type_options = []
+    try:
+        config_list = ctx.compendium_service.get_charge_plan_mission_type_list('恶名狩猎')
+        mission_type_options = [{'label': c.label, 'value': c.value} for c in config_list]
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)